tests/models/test_game.py

```diff
@@ -1,13 +1,20 @@
-from datetime import datetime
-
 import pytest
-from mars_api.models import Game
 
+from mars_api.models import MAPS, Game
+
+pytestmark = pytest.mark.django_db
+
+
+def test_game_cannot_be_created_with_invalid_game_map():
+    pytest.fail("unimplemented")
+
+
+def test_game_returns_players_nicknames():
+    """When PlayerScores exist, check that corresponding Player nicknames are returned."""
+
+    # 1. mock PlayerScores. Sidenote: PlayerScores should be allowed to be created without an existing Player.
```

Comment:
Very true - the sidenote.

Reply:
Though it wouldn't be correct to call those objects mocks - they are the same instances you would have if you created them manually.